### PR TITLE
fixed link

### PR DESCRIPTION
**Pull request by Rick-Anderson**
broken by mvc merge into aspnet
I made no changes to Simon Timms Blog:
but I don't see the diff in the diff view.

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@danroth27 @ardalis  please review

**Comment by danroth27** [comment]
:shipit:

**Comment by Rick-Anderson** [comment]
@danroth27 I'll change the link from RC1 to master first - right?

File: aspnet/mvc/views/dependency-injection.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ Sections:
 	- `Populating Lookup Data`_
 	- `Overriding Services`_
 
-`View sample files <https://github.com/aspnet/Docs/tree/1.0.0-rc1/mvc/views/dependency-injection/sample>`_
+`View sample files <https://github.com/aspnet/Docs/tree/1.0.0-rc1/aspnet/mvc/views/dependency-injection/sample>`_
```
Comment (danroth27) [inline aspnet/mvc/views/dependency-injection.rst]: Please point this link to master instead of the rc1 branch.

**Comment by danroth27** [comment]
Yup. Just push another commit with the update.

**Pull request closed without merging**